src/tscore/Layout.cc

```diff
@@ -155,7 +155,8 @@ Layout::Layout(std::string_view const _prefix)
     std::string path;
     int len;
     if (getenv("TS_ROOT") != nullptr) {
-      std::string env_path(getenv("TS_ROOT"));
+      const char *const env = getenv("TS_ROOT");
+      std::string env_path(nullptr != env ? env : "");
```

Comment:
(just curious) This `env` can't be nullptr in this `if` block. A fails positive of clang-analyzer? or it says something else?